dandi/organize.py

```diff
@@ -1151,7 +1169,12 @@ def validate_organized_path(
             errors.append(
                 ValidationResult(
                     id="DANDI.METADATA_MISMATCH_SUBJECT",
-                    origin=ValidationOrigin(name="dandi", version=__version__),
+                    origin=Origin(
+                        type=OriginType.VALIDATION,
+                        validator=Validator.dandi,
+                        validator_version=__version__,
+                        standard=Standard.DANDI_LAYOUT,
+                    ),
```

Comment:
3 "uses" for the same `origin` . Similarly to other similar cases -- just define `origin` a tthe top of the function and reuse here?  would make code shorter and thus easier to read.

But then -- this and other functions will be invoked in a loop across paths. So I wonder if we should centrally define some of those origins, like here `origin_dandi_layout` and alike? this would save some cycles/bytes across files. WDYT?

- we should at least define / reuse